xmtp_api_grpc/src/grpc_client/native.rs

```diff
@@ -0,0 +1,134 @@
+use crate::error::GrpcBuilderError;
```

Comment:
it looks like my cherry-pick pulled these in b/c the api builder was modified which req. changing the `build()` from await to sync of these files. all the bindings are using the right client (from `grpc_api_helper`) so this should be OK. I can re-pull and re-cherry-pick if we want to be cleaner, though